pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "daboizzz-discord-bot"
+version = "0.1.0"
+description = ""
+authors = ["Alex Karet <apk115@ic.ac.uk>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+"discord.py" = {git = "https://github.com/Rapptz/discord.py"}
+dnspython = "1.16.0"
+PyNaCl = "1.3.0"
+async-timeout = "3.0.1"
+discord-py-slash-command = "^1.1.0"
+python-dotenv = "^0.17.0"
+
+[tool.poetry.dev-dependencies]
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
```

Comment:
I set up poetry, you can adjust the names here should you want to use it going forward.